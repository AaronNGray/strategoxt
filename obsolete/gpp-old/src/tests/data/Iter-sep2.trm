a b A b B C D e E F g
