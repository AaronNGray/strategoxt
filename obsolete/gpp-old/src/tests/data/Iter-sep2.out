HV([],[H([SOpt(HS,"0")],[V([],[S("a"),S("b")]),S("A")]),H([SOpt(HS,"0")],[V([],[S("b")]),S("B"),S("C"),S("D")]),H([SOpt(HS,"0")],[V([],[S("e")]),S("E"),S("F")]),S("g")])
